components/CommentCount/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+class CommentCount extends React.Component {
+  formatString(count) {
+    if(count === 1) {
+      return `${count} comment`
```

Comment:
I originally looked for a library when doing mine but as it was such a small implementation I thought it was more work to implement the library than to not implement it TBH